SimMuon/GEMDigitizer/src/GEMSignalModel.cc

```diff
@@ -0,0 +1,162 @@
+#include "SimMuon/GEMDigitizer/interface/GEMSignalModel.h"
+#include "Geometry/GEMGeometry/interface/GEMEtaPartitionSpecs.h"
+#include "Geometry/CommonTopologies/interface/TrapezoidalStripTopology.h"
+#include "Geometry/GEMGeometry/interface/GEMGeometry.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "CLHEP/Random/RandFlat.h"
+#include "CLHEP/Random/RandPoissonQ.h"
+#include "CLHEP/Random/RandGaussQ.h"
+#include "CLHEP/Units/GlobalPhysicalConstants.h"
+#include "DataFormats/Math/interface/GeantUnits.h"
+#include <cmath>
+#include <utility>
+#include <map>
+
+GEMSignalModel::GEMSignalModel(const edm::ParameterSet& config)
+    : GEMDigiModel(config),
+      averageEfficiency_(config.getParameter<double>("averageEfficiency")),
+      averageShapingTime_(config.getParameter<double>("averageShapingTime")),
+      timeResolution_(config.getParameter<double>("timeResolution")),
+      timeJitter_(config.getParameter<double>("timeJitter")),
+      signalPropagationSpeed_(config.getParameter<double>("signalPropagationSpeed")),
+      digitizeOnlyMuons_(config.getParameter<bool>("digitizeOnlyMuons")),
+      resolutionX_(config.getParameter<double>("resolutionX")),
+      muonPdgId(13),
+      cspeed(CLHEP::c_light),
```

Comment:
Personally I prefer to have a common source for physical constants. The hardcoded definitions can easily become inconsistent.

Reply:
@kpedro88 I may agree, but we were trying to clean the use of CLHEP for units of measurements, and here we are keeping it. At least for some very basic constants we should consider whether we may provide something, or think to other possible sources that we have (e.g. https://root.cern/doc/v618/namespaceTMath.html#a22dbfc37d5e400dec5ac6b1351e2061a )

Reply:
@fabiocos: I think the plan was to keep using CLHEP for physics constants. Those are separate from the migration away from the CLHEP unit conversions.
Since we will keep using CLHEP for random numbers, the idea was to keep using the CLHEP physics constants also. As long as it is remembered that the CLHEP constants use the CLHEP/Geant4 units convention, there's no strong reason not to use them.

Reply:
@cvuosalo the CLHEP physical constants are just using CLHEP units in their definition, so we just keep using CLHEP units but "behind the scene"